src/components/DailyScan/DailyScanWizard.tsx

```
@@ -0,0 +1,448 @@
+            <CameraCapture
+              angle={currentAngle}
+              onCapture={handlePhotoCaptured}
+              onCancel={() => setCurrentStep('intro')}
+            />
+          </div>
+        );
+
+      case 'upload':
+        const totalSize = getTotalPhotoSize(photoBlobs);
+
+        return (
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)